support-frontend/assets/pages/paper-subscription-landing/paperSubscriptionLandingPage.tsx

```diff
@@ -45,7 +45,9 @@ const pageQaId = 'qa-paper-subscriptions';
 function PaperLandingPage({
 	productPrices,
 	promotionCopy,
+	participations,
 }: PaperLandingPropTypes) {
+	const hideDigital = participations.newProduct === 'variant' ? true : false;
```

Comment:
```suggestion
	const hideDigital = participations.newProduct === 'variant';
```

Reply:
Of course, will neaten a bit further now, thanks for the review!

Reply:
All done, if you get a minute, etc etc, ta